tests/test_main.cpp

```diff
@@ -60,6 +60,11 @@
 #include "test_validate_testing.h"
 #include "test_variant.h"
 
+#include "modules/modules_enabled.gen.h"
+#ifdef MODULE_REGEX_ENABLED
+#include "test_regex.h"
+#endif
```

Comment:
This should actually go in `modules/regex/tests/` as it's module-specific. The `modules/modules_tests.gen.h` include below is used to include those.

Reply:
Alright, thanks for your quick reply!

If you wonder, I converted it to a 'Draft' due to the formatting issues. Hope that didn't confuse you while reviewing.